backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java

```diff
@@ -24,7 +24,7 @@
 @Slf4j
 public class FileUploadController {
   public static final String TEXT_CSV_CONTENT_TYPE = "text/csv";
-  private final UploadService uploadService;
+  private final PatientBulkUploadService patientBulkUploadService;
```

Comment:
Thanks for changing the name of this service, it's been bugging me since its inception 😂

Reply:
Yeah, it was getting way to confusing to switch between "UploadService" and "TestResultUploadService" 😆